util/testutil/util.go

```diff
@@ -137,3 +137,7 @@ func RandomDigest() string {
 	d := digest.FromBytes(RandomByteData(10))
 	return d.String()
 }
+
+func TrimSha256Prefix(s string) string {
```

Comment:
nit: I think this is the same as `Digest.Parse(s).Encoded()`

Reply:
It is, but `Digest.Parse()` requires error handling and `TrimSha256Prefix` is used inline in multiple places.
This function is now new to this PR, just moved. I'm happy to refactor the call sites to use `...Encoded()` and handle any errors.